### PR TITLE
feat: publish v22 params

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

File: filecoin-proofs/parameters.json [hunk 1]
```diff
@@ -1,102 +1,57 @@
...
+  "v22-stacked-proof-of-replication-PoseidonHasher-Sha256Hasher-97898c09be471d01d125f93ba4dd8d1a5634e404c1c7d1ea21014d0c8656a29a.vk": {
+    "cid": "Qmc9tXAG99A8ykiV5AwXenKTfd2GXtLA6RdWYyhyT9J6c8",
+    "digest": "48c207aa03b9db61ed595b02ff380d80",
     "sector_size": 1073741824
   },
-  "v20-stacked-proof-of-replication-e99a585174b6a45b254ba4780d72c89ad808c305c6d11711009ade4f39dba8e9.vk": {
-    "cid": "QmWReGfbuoozNErbskmFvqV4q36BY6F2WWb4cVFc3zoYkA",
-    "digest": "20d58a3fae7343481f8298a2dd493dd7",
-    "sector_size": 1073741824
+  "v22-stacked-proof-of-replication-PoseidonHasher-Sha256Hasher-b9678fc0c28dfcec342ca4c7886020d583b16258c9bc93aafad679d09400e401.vk": {
```
Comment (laser) [inline filecoin-proofs/parameters.json]: BLOCKING: This changeset is missing the v22 parameter file for 1KiB sectors. It contains the verifying key, but not the corresponding `.params` file.

Comment (laser) [reply filecoin-proofs/parameters.json]: I'm going to remove my block on this PR, as you'll likely fix the issue before I wake up.

**Comment by dignifiedquire** [comment]
it seems there are quite a few files missing, sth failed in parampublish from the looks of it

File: filecoin-proofs/parameters.json (hunk 1 above)
Comment (dignifiedquire) [reply filecoin-proofs/parameters.json]: should be fixed now

**Comment by laser** [comment]
you the man

**Pull request merged**